vavr/src/main/java/io/vavr/collection/AbstractQueue.java

```diff
@@ -164,9 +164,16 @@ public Q removeAll(Iterable<? extends T> elements) {
         return Collections.removeAll((Q) this, elements);
     }
 
-    @SuppressWarnings("unchecked")
+    @Deprecated
```

Comment:
It is good to mark it! Maybe we can remove the method for Vavr 1.0.0 - but for now it can stay as-is. Thx!